services/frontend-service/src/ui/components/ServiceLane/ServiceLane.tsx

```diff
@@ -14,20 +14,38 @@ You should have received a copy of the GNU General Public License
 along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
 
 Copyright 2021 freiheit.com*/
-import { useDeployedReleases } from '../../utils/store';
+import { useDeployedReleases, useReleases } from '../../utils/store';
 import { ReleaseCard } from '../ReleaseCard/ReleaseCard';
 import { Button } from '../button';
 import { DeleteWhite, HistoryWhite } from '../../../images';
-import { Application } from '../../../api/api';
+import { Application, Release } from '../../../api/api';
+
+function getReleaseDiff(releases: Array<Release>, version1: number, version2: number) {
```

Comment:
```suggestion
function getNumberOfReleasesBetween(releases: Array<Release>, version1: number, version2: number) {
```
suggestion to rename function

Reply:
I would also rename "version1" to "lowerVersion" and 2 to "higherVersion" or something like that

Reply:
done